maketest.sh

```diff
@@ -29,3 +35,29 @@ else
 	$MAKE $@
 fi
 
+# For now (while this code is being prototyped) I only want this to be
+# done while running in jenkins, to avoid a potentially destabilising
+# change if people are running other java processes on their local system
+set -x
+if [ ! -z "$EXECUTOR_NUMBER" ]; then
+  if [ "$OSTYPE" = "cygwin" ]; then
+    # Windows code based on https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1669#issuecomment-727863096
+    echo PROCESSCATCH: Showing java processes - one will be the jenkins agent - I will remove any of these created since $JOBSTARTTIME:
+    powershell -c "Get-Process java"
+    C=`powershell -c "(Get-Process java | select id,processname,starttime | where {\\$_.StartTime -gt (Get-Date -Date \"$JOBSTARTTIME\")} | measure).count" | tr -d \\\\r`
+    if [ $C -ne 0 ]; then
+      echo PROCESSCATCH: I am going to attempt to stop the following ones:
+      powershell -c "Get-Process java | select id,processname,starttime | where {\$_.StartTime -gt (Get-Date -Date \"$JOBSTARTTIME\")}"
+      powershell -c "Get-Process java | select id,processname,starttime | where {\$_.StartTime -gt (Get-Date -Date \"$JOBSTARTTIME\")} | stop-process"
+      echo PROCESSCATCH: Removed processes - here is what is remaining:
+      powershell -c "Get-Process java"
+    fi
+  else
+    # Non-Windows code from https://ci.adoptopenjdk.net/job/SXA-processCheck
+    echo PROCESSCATCH: Checking for any leftover java processes on the machine
+    if ps -fu $USER | grep java | egrep -v "remoting.jar|agent.jar|grep"; then
+      echo PROCESSCATCH: There are rogue procesess still on the machine
+      echo PROCESSCATCH: Not killing them for now as this script should be tested more thoroughly first
```

Comment:
Not sure you need the colour commentary of 'script should be tested more thoroughly first' ... essentially if you get into this block, it is an alert that should trigger someone to manually clean-up the rogue processes

Reply:
Was more of a "TODO" (Although SXA-processCheck covers that situation)_ ... May just edit it to say that as a comment along with the reference to the TKG issue